plugins/graphql-java-codegen-gradle-plugin/graphql-codegen-gradle-plugin/build.gradle

```diff
@@ -0,0 +1,48 @@
+plugins {
```

Comment:
Should this file be one level up? There is an empty `build.gradle` one level up. Is it normal?

Reply:
I think it is totally normal. This set of plugins are required only by `graphql-codegen-gradle-plugin` module.